app/models/user/UserRoleTable.scala

```diff
@@ -51,4 +51,19 @@ object UserRoleTable {
     val commServ: Boolean = communityService.getOrElse(currRole.map(_.communityService).getOrElse(false))
     userRoles.insertOrUpdate(UserRole(currRole.map(_.userRoleId).getOrElse(0), userId.toString, newRole, commServ))
   }
+
+  /**
+    * Gets the community service status of the user.
+    */
+  def getCommunityService(userId: UUID): Boolean = db.withSession { implicit session =>
+    userRoles.filter(_.userId === userId.toString).map(_.communityService).first
+  }
+
+   /**
+    * Sets the community service status of the user.
+    */
+  def setCommunityService(userId: UUID, newCommServ: Boolean): Int = db.withTransaction { implicit session =>
+    val currRole: Option[UserRole] = userRoles.filter(_.userId === userId.toString).firstOption
+    userRoles.insertOrUpdate(UserRole(currRole.map(_.userRoleId).getOrElse(0), userId.toString, currRole.map(_.roleId).getOrElse(0), newCommServ))
```

Comment:
An entry in the `user_role` table should be made when the account is created. Let's try and keep it so that created those only happens in one place, so let's not do `insertOrUpdate` here, let's just update an entry if it exists.